packages/bindings/src/querier.rs

```diff
@@ -154,7 +154,7 @@ impl<'a> ProvenanceQuerier<'a> {
     /// }
     ///
     /// // Text with timestamp.
-    /// #[derive(Serialize, Deserialize, Clone, Debug, PartialEq, JsonSchema)]
+    /// #[derive(Serialize, Deserialize, Clone, Debug, Eq, PartialEq, JsonSchema)]
```

Comment:
Comment is out of date, should use the new derive.

Reply:
the bindings package actually remains unchanged. only inputs/outputs of the contracts use the new derived schema